cypress/integration/team/teams_spec.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Teams Suite', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('Cancel out of leaving a team', () => {
+        // * check the team name
+        cy.get('#headerTeamName').should('contain', 'eligendi');
+
+        // 2. open the drop down menu
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+        // 3. click the leave team
+        cy.get('#sidebarDropdownMenu #leaveTeam').should('be.visible').click();
+
```

Comment:
Please add element ID to leave team modal (e.g. `leaveTeamModal` [here](https://github.com/mattermost/mattermost-webapp/blob/eef36a3149f37cb59762f8fbb023e097aa3e0905/components/leave_team_modal/leave_team_modal.jsx#L80)) so we could add validation that the leave team modal has opened up after clicking above, like:
```javascript
// * Check that the "leave team modal" opened up
cy.get('#leaveTeamModal').should('be.visible');
```